gdscript-customization.el

```diff
@@ -129,6 +129,16 @@ so it is not slowing down Godot execution."
   :type 'boolean
   :group 'gdscript)
 
+(defcustom gdscript-docs-force-online-lookup nil
+  "the string to determine if we are browsing online API to allow filtering in EWW"
+  :type 'boolean
+  :group 'gdscript)
+
+(defcustom gdscript-docs-root-match "docs.godotengine"

```

Comment:
Should this be a customizable property? It's an internal value the package uses to filter pages it downloads from the docs. If you use local docs, we can also detect it and avoid the need for a domain check to run the filter.

Reply:
It is used in local docs. But happy to remove the customisation.